assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -66,10 +68,12 @@ function PaddingControls({ selectedElements, pushUpdateForObject }) {
           horizontal: commonPadding,
           vertical: commonPadding,
         };
+
+        if (padding.locked === MULTIPLE_VALUE) update.locked = lockPadding;
```

Comment:
Nit: use `{}`. makes a lot easier to read.

/cc @swissspidy I thought we had a lint for this?

Reply:
Actually, I believe this line is not quite correct. I think, once the padding is edited, we just want to set it two lines above to always be `true`. E.g. 

```
update = {
  horizontaol: commonPadding,
 ...
  locked: true
}
```

Reply:
Also, I think this particular test is missing.

Reply:
Removed and added locked to above. And add covering tests for different lock ratio elements combination, the lock ratio should be false. And in this case we will have locked sets for only true cases 😄

Reply:
@dvoytenko we did have a lint for this, but apparently `eslint-config-prettier` disables it and requires it to be set explicitly due to potential formatting conflicts. PR imminent...